superset/config.py

```diff
@@ -748,7 +748,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 #
 # Note if no end date for the grace period is specified then the grace period is
 # indefinite.
-SIP_15_ENABLED = False
+SIP_15_ENABLED = True
 SIP_15_GRACE_PERIOD_END: Optional[date] = None  # exclusive
```

Comment:
Should we add in a grace period date here? Maybe `2020-03-31` or something like that?

Reply:
@etr2460 I feel like indefinite is preferred to having a hard date, it would be troublesome if someone deployed this change and weren't aware that the grace period had already finished and thus this feature would serve as a hard transition.

Reply:
@mistercrunch @willbarrett do we want to take a passive (no end date) or aggressive (end date specified) approach.

Reply:
I'm not speaking for @mistercrunch , but I would recommend the aggressive approach. That would provide us a milestone for when the flag can be removed.